Remotion/ObjectBinding/Web/UI/Controls/BocList.RowMenuSupport.cs

```diff
@@ -80,6 +94,52 @@ private void EnsureRowMenusInitialized ()
       _rowMenus = rowMenus.ToArray();
     }
 
+    private bool TryGetRowMenu (string targetInfo, [NotNullWhen(true)] out string? id)
+    {
+      var rowMenuIDPrefix = UniqueID + c_rowMenuIDPrefix;
+      if (targetInfo.StartsWith(rowMenuIDPrefix))
+      {
+        id = targetInfo.Substring(rowMenuIDPrefix.Length);
+        return true;
+      }
+
+      id = null;
+      return false;
+    }
+
+    private void EnsureRowMenuInitialized (string id)
+    {
+      if (_rowMenusInitialized)
+        return;
+
+      _singleRowMenuInitialized = true;
```

Comment:
Great question: I don't think it matters, as this is merely used for answering things in postbacks. There might be an issue if the same row menu gets initialized twice (same id), but I'm not certain that that would actually problematic. 

We might want to have a list (or dictionary) of created initialized rows instead of this boolean, and check for its emptiness instead.

@MichaelKetting any input?